docs/nuxt/lib/site.js

```
@@ -32,13 +32,39 @@ const SITE_DESCRIPTION
+  explanation: {
+    label: 'Concepts',
+    description: 'How Druxt works and why: architecture, routing, the store, schemas and component resolution.',
+    priority: 0.7,
+    changefreq: 'monthly',
+  },
+  components: {
+    label: 'Components',
+    description: 'Every Druxt component, generated from the package source by druxt-docgen.',
+    priority: 0.5,
+    changefreq: 'yearly',
+  },
```

Biome (2.5.8)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)